core/backend/src/test/standalone/ChangesetReader.test.ts

```diff
@@ -0,0 +1,205 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert, DbOpcode, DbResult } from "@bentley/bentleyjs-core";
+import { IModelJsNative } from "@bentley/imodeljs-native";
+import { IModelHost } from "../../imodeljs-backend";
+import { KnownTestLocations } from "../KnownTestLocations";
+import * as path from "path";
+
+// spell-checker: disable
+
+describe("changeset reader", () => {
+  it("read data using getColumnValue()", () => {
+    const changesetFile = path.join(KnownTestLocations.assetsDir, "changesets/data.changeset");
+    const reader: IModelJsNative.ChangesetReader = new IModelHost.platform.ChangesetReader();
+    assert(reader.open(changesetFile, false) === DbResult.BE_SQLITE_OK);
+
+    assert(reader.step() === DbResult.BE_SQLITE_ROW);
```

Comment:
Why are using `assert` from bentleyjs-core instead of chai's `assert` or `expect`?

Reply:
Did not realized i was using that one. I fixed it.